File: app/src/main/java/com/tomatogaming/mealmelt/Model/Recipe.java
package com.tomatogaming.mealmelt.Model;

import java.util.List;

public class Recipe
{
    public int Id = 1;
    public String Name = "Test";
    public String Description = "Test";
    public com.tomatogaming.mealmelt.Model.Author Author;
    public com.tomatogaming.mealmelt.Model.Category Category;
    public List<Recipe> Sides;
    public List<Step> Steps;
    public List<Ingredient> Ingredients;
    public List<Tool> Tools;
    public int PhotoId;
}

File: MealMelt.Repository/Models/Recipe.cs
namespace MealMelt.Repository.Models
{
    public class Recipe
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Author { get; set; }
        public Category Category { get; set; }
        public int? PhotoId { get; set; }
    }
}